src/test/java/ProjectOcean/Model/CoursePlanningSystemTests.java

```diff
@@ -10,161 +10,134 @@
         String actual = coursePlanningSystem.getStudyPeriod(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getExaminatorTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-
-        String expected = courses.get(courseID).getExaminator();
+        String expected = courseID.getExaminator();
         String actual = coursePlanningSystem.getExaminator(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getExaminationMeansTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-
-        String expected = courses.get(courseID).getExaminationMeans();
+        String expected = courseID.getExaminationMeans();
         String actual = coursePlanningSystem.getExaminationMeans(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getLanguageTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-
-        String expected = courses.get(courseID).getLanguage();
+        String expected = courseID.getLanguage();
         String actual = coursePlanningSystem.getLanguage(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getRequiredCoursesTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-        List<UUID> expected = new ArrayList<>();
-        Iterator iterator = courses.get(courseID).getRequiredCourses().iterator();
+        List<String> expected = new ArrayList<>();
 
-        while (iterator.hasNext()){
-            expected.add(((Course)iterator.next()).getId());
+        for (String string : courseID.getRequiredCourses()) {
+            expected.add(string);
         }
 
-
-        List<UUID> actual = coursePlanningSystem.getRequiredCourses(courseID);
+        List<String> actual = coursePlanningSystem.getRequiredCourses(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getCoursePMLinkTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-
-        String expected = String.valueOf(courses.get(courseID).getCoursePMLink());
+        String expected = String.valueOf(courseID.getCoursePMLink());
         String actual = coursePlanningSystem.getCoursePMLink(courseID);
 
         Assert.assertEquals(expected, actual);
     }
 
     @Test
     public void getCourseDescriptionTest(){
+        ICourse courseID = courses.get(0);
 
-        UUID courseID = allUUIDs.get(0);
-
-        String expected = String.valueOf(courses.get(courseID).getCourseDescription());
+        String expected = String.valueOf(courseID.getCourseDescription());
         String actual = coursePlanningSystem.getCourseDescription(courseID);
 
         Assert.assertEquals(expected, actual);
```

Comment:
I want none of these tests here, because I dont want the methods in course planning to begin with, make a test of course instead with all of these tests